soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
I mean pivot works. I fixed that. Its just MotionControl is still the mess its always been with our rotation stuff.